DESCRIPTION

```diff
@@ -0,0 +1,23 @@
+Package: IOHexperimenter
+Type: Package
+Title: Benchmarking Part of the 'IOHprofiler'
+Version: 0.0.0.9000
+Author: Furong Ye [cre, aut], Hao Wang [aut], Diederick Vermetten [aut], Carola Doerr [aut], Thomas Bäck [aut]
+Maintainer: Furong Ye <f.ye@liacs.leidenuniv.nl>
+Description: The benchmarking module for the Iterative Optimization Heuristics
+    Profiler ('IOHprofiler'). This module provides benchmarking in the 'IOHprofiler'
+    format, which can be visulized using the 'IOHanalyzer' module.
+License: BSD_3_clause + file LICENSE
+Encoding: UTF-8
+LazyData: true
```

Comment:
This means the lazy loading of the data set, if there is any, right?

Reply:
Yes, this seems to be highly encouraged when creating a package